explorer/BUILD

```diff
@@ -86,9 +153,22 @@ file_test(
             "testdata/linked_list/typed_linked_list.carbon",
         ],
     ),
-    deps = [":file_test_common"],
 )
 
+[
+    sh_test(
+        name = "%s_%s" % (test_dir, test_target),
+        size = "small",
+        srcs = ["explorer_tests"],
+        args = [
+            "--no_file_tests",
+            "--gtest_filter=" + test_suite + ".*",
+            "--fail_if_no_tests_run",
```

Comment:
(Historical, since I've removed this again, but: this is a limitation of the absl Flags library. You can't nicely have an optional flag that distinguishes between the flag being absent and being present with an empty value, because the library determines whether the flag is present by checking whether its string value differs from the default. There are workarounds like setting a default of "none" rather than an actually empty default, but that seemed too ugly.)